package.json

```diff
@@ -34,6 +35,9 @@
       "mocha"
     ],
     "esnext": true,
-    "space": true
+    "space": true,
+    "rules": {
+      "linebreak-style": 0
```

Comment:
Are you on `windows`? What error did it cause w/o this? Probably I should add a `.gitattribute` to the repo.